### PR TITLE
IN-207 update README.md to document the script-logger

### PR DESCRIPTION
**Pull request by sberczukTS**
Once approved I'll move the version tag rather that rev it, unless anyone objects.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,10 +11,10 @@ You can read more about the [transports](#transports) and [features](#features)
 // recommended
 yarn add git://github.com/tetrascience/ts-lib-logger.git#^1.0.0 --production
 
-npm install tetrascience/ts-lib-logger --save
+yarn install tetrascience/ts-lib-logger --save
```
Comment (chunfengd) [inline README.md]: I think this should be `yarn add tetrascience/ts-lib-logger`. usually we also need tags, like this: `yarn add tetrascience/ts-lib-logger#^1.2.0`

File: README.md [hunk 2]
```diff
@@ -104,6 +105,24 @@ Read about util.inspect [here](https://nodejs.org/api/util.html#util_util_inspec
...
+* A Timestamp, with the key `timestamp`
+
+
+```javascript
+// Script Code
+const logger = require('ts-lib-logger')('script');
+logger.info("Processing file");
+```
+
+The consumer (for example ts-service-task-runner:models/task.js) will parse the
```
Comment (chunfengd) [inline README.md]: This seems to be an unfinished sentence. :P

**Comment by mdumic** [comment]
Generally we allow IntelliJ* files in our projects, but not all of them. See here:
https://github.com/tetrascience/ts-service-pipeline/blob/master/.gitignore#L60

File: README.md (hunk 1 above)
Comment (sberczukTS) [reply README.md]: I'll follow your lead here. I just replaced npm with yarn. wouldn't the yarn install just work off of the json config?

File: README.md (hunk 2 above)
Comment (sberczukTS) [reply README.md]: * details left to the reader* :) Sorry about that, will fix.

File: .gitignore [hunk 3]
```diff
@@ -1,2 +1,10 @@
+# IntelliJ and Webstorm files
 .idea
```
Comment (sberczukTS) [inline .gitignore]: I can put .iml files back but if we are excluding project files, do we want to allow module files?

Comment (sberczukTS) [reply .gitignore]: After talking with @joegoldbeck and @mdumic keeping .iml in .gitignore is the way to go.

**Pull request merged**